lib/src/pages/route_initializer.dart

```diff
@@ -54,6 +54,15 @@ class RouteInitializerState extends State<RouteInitializer> {
 		try {
 			if (!Services.instance.isReady) {
 				await Services.instance.init();
+
+				final brightness = Services.instance.prefs.brightness;
+				ThemeChanger.of(context).brightness = caseConverter<Brightness> (
+					value: brightness,
+					onTrue: Brightness.light,
+					onFalse: Brightness.dark,
+					onNull: MediaQuery.of(context).platformBrightness,
+				);
+				
 			}
```

Comment:
```suggestion

				ThemeChanger.of(context).brightness = caseConverter<Brightness> (
					value: Services.instance.prefs.brightness,
					onTrue: Brightness.light,
					onFalse: Brightness.dark,
					onNull: MediaQuery.of(context).platformBrightness,
				);
			}
```

You can merge the two together like this